refinery/core/admin.py

```diff
@@ -151,7 +151,7 @@ class SiteProfileAdmin(GuardedModelAdmin):
 
 
 class SiteStatisticsAdmin(AdminFieldPopulator):
-    pass
+    change_list_template = "admin/core/sitestatistics/change_list.html"
```

Comment:
Is folder necessary for a single file?

Reply:
I thought it made sense for organizational purposes